provider/standalone/standalone.go

```diff
@@ -33,6 +33,9 @@ func (p *StandaloneProvider) ConfigureRun(state *run.State) error {
 	state.WorkerPoolID = pc.WorkerPoolID
 	state.WorkerGroup = pc.WorkerGroup
 	state.WorkerID = pc.WorkerID
+	state.WorkerLocation = map[string]string{
+		"cloud": "standalone",
+	}
```

Comment:
For this and the `standalone` provider, I believe users will want to be able to specify their own location data in the worker-runner config file.  For example, releng probably wants to be able to put `"datacenter": "mdc1"` or something like that in the data structure.